### ISSUE TITLE
Make TT agent realise if the time frame for something happen is reasonable

### ISSUE THREAD

**Issue by kongzii** [gnosis/prediction-market-agent#296]
Following up on this issue https://github.com/gnosis/prediction-market-agent/issues/292.

It seems that agent doesn't really gasp the fact that 1 day to find a person isn't much, and probability should be a lot lower than the given 10%.

[This market](https://presagio.pages.dev/markets?id=0xec2d18818ae6311e113ae115027c1b12c3f4e3e5) was created 2 days ago and will be closed in 1 day.

The issue above added to the agent this information:

> Event date is 2024-07-03 00:00:00+00:00 and 1 days remaining. Market is already open for 2 days.

But that doesn't trigger in the agent the fact that 1 day isn't enough to find her.

It would be nice if the agent would google up this and added into into the prediction prompt:

> Where was Ruja Ignatova last seen?
> Ignatova was last spotted Oct. 25, 2017

Now, if the agent would see that she is wanted for 7 years and there is only 1 day remaining to find her, it should hopefully drive the probability for it happening to almost 0%.

Open question is, how to implement a logic like this in the TT agent, without overfitting his decision-process into this particular market.

**Comment by gabrielfior** [comment]
@kongzii can we close this after #295 is merged?

**Comment by gabrielfior** [comment]
> No, logic wanted from this issue is more complex, hopefully will work better than #295.

@kongzii  It would be nice to have some implementation suggestion, to better showcase the complexity. Could you please add some more info?
